File: mario-more.c
# include <stdio.h>
# include <cs50.h>

int main()
{
    int n;
    // Prompting user for input
    do 
    {
        n = get_int("Enter positive number: ");
    } 
    while(n<=0 || n>8);
    // Using nested for loop to make the bricks
    for (int i=1; i<=n; i++)
    {
        //Used for loop to make empty spaces
        for (int j=n-i; j>0; j--)
        {
            printf(" ");
        }
        //Using loop to make 1st pattern.
        for (int k=1; k<=i; k++)
        {
            printf("#");
        }
        // Making double empty spaces between the patter
        printf("  ");
        // Using loop to make 2nd pattern
        for (int l=1; l<=i; l++)
        {
            printf("#");
        }
        // printing a new line
        printf("\n");
    }
    
}

File: mario-more.py
# Importing cs50 library
import cs50 as c
# Getting input from the user
height = 0
while height > 8 or height < 1:
    height = c.get_int("Height: ")
# Using for loop to make the pattern
for i in range(height):
    for k in range(height-i, 1, -1):
        print(" ", end='')
    for j in range(i+1):
        print("#", end='')
    print("  ", end='')
    for l in range(i+1):
        print("#", end='')
    print()